packs/data/strength-of-thousands-bestiary.db/ajbal-kimon.json

```
@@ -882,7 +881,30 @@
                 "requirements": {
                     "value": ""
                 },
-                "rules": [],
+                "rules": [
+                    {
+                        "definition": [
+                            "item:base:trident",
+                            "item:melee"
+                        ],
+                        "key": "AdjustStrike",
+                        "mode": "add",
+                        "property": "weapon-traits",
+                        "value": "shove"
+                    },
+                    {
+                        "key": "Note",
+                        "outcome": [
+                            "criticalSuccess"
+                        ],
+                        "predicate": [
+                            "item:base:trident"
+                        ],
+                        "selector": "strike-attack-roll",
+                        "text": "PF2E.NPCAbility.AjbalKimon.TridentBullyNote",
```

```suggestion
                        "text": "{item|system.description.value}",
```
We don't need to localize stuff just to chop off a tiny bit of the description.